llvm/include/llvm/Analysis/ScalarEvolution.h

```diff
@@ -1044,6 +1044,11 @@ class ScalarEvolution {
   bool isKnownToBeAPowerOfTwo(const SCEV *S, bool OrZero = false,
                               bool OrNegative = false);
 
+  /// Check that memory access offsets in S are multiples of M.  Assumptions
+  /// records the runtime predicates under which S is a multiple of M.
```

Comment:
```suggestion
  /// Check that memory access offsets in \p S are multiples of \p M.  \p Assumptions
  /// records the runtime predicates under which \p S is a multiple of \p M.
```

I also think that, if I understand it correctly, `S` no longer needs to be a memory access.

Reply:
Done.